_typos.toml

```diff
@@ -0,0 +1,13 @@
+# Files for typos
+# Instruction:  https://github.com/marketplace/actions/typos-action#getting-started
+
+[default.extend-identifiers]
+
+[default.extend-words]
+# Don't correct following words
+nin="nin"
```

Comment:
Could you link a line where this appears please?

Reply:
Hereis the result of `typos` without the line `nin="nin"`.
https://gist.github.com/shirayu/43649cc2a3b32c875c40027c02b0c946

At first I thought `nin` was an abbreviation for something, but is it a typo for `min`?

Reply:
Looking at https://github.com/huggingface/diffusers/blame/5dda1735fda047f4242d28f91e6e457b9760d52d/src/diffusers/models/resnet.py#L325 
I think `use_in_shortcut` was meant here @patrickvonplaten?

Reply:
I fixed `use_in_shortcut` at b75558b.

```diff
-        new_item = new_item.replace("nin_shortcut", "conv_shortcut")
+        new_item = new_item.replace("in_shortcut", "conv_shortcut")
```

Please check whether this change is OK.

Reply:
Another use of `MIN` is [here](https://gist.githubusercontent.com/shirayu/43649cc2a3b32c875c40027c02b0c946/raw/44e2be398f73f72cb5971113754c2e4b85e7c177/NIN2).

I think this should not be changed.
I added a memo about this at b7845a358f580118bf7da6c1c17ec2fa10b6ca50.